scripts/utils/checkSetup.R

```diff
@@ -8,7 +8,10 @@ stopifnot(`pdflatex not found, check your LaTeX installation` = Sys.which("pdfla
 message("checking for pdflatex executable on your PATH - ok")
 
 message("checking if required R packages are installed")
-missingDeps <- Filter(function(x) !requireNamespace(x, quietly = TRUE),
+filter_func <- function(x) {
+  return(!requireNamespace(x, quietly = TRUE) && x != "R")
+}
+missingDeps <- Filter(filter_func,
                       renv::dependencies(dev = TRUE)[, "Package"])
```

Comment:
```suggestion
missingDeps <- Filter(function(x) !requireNamespace(x, quietly = TRUE),
                      setdiff(renv::dependencies(dev = TRUE)[, "Package"], "R"))
```

Reply:
`setdiff` also applies `unique`, so no package is checked twice. There is no linter in remind (yet), but I'd prefer if we could stick to camel case anyway, so I'd suggest renaming `filter_func` to `filterFunc` if you want to stick to that solution.

Reply:
`setdiff` here works nicely as well. For me, `x != "R"` is a bit more readable, but that's probably just because I am *always* confused when seemingly scalar values in R are treated as sets or vectors. How can "R" be a set, it is just a single string! :wink: